src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -167,14 +155,25 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
       eventTimeAndOrderComparator
     );
 
+    const currentVenueBookMarkEvents = eventsFilledWithPriority.filter(
+      (event) =>
+        event.isSaved && event.venueId === currentVenue?.id?.toLowerCase()
```

Comment:
If possible
```
const currentVenueId = currentVenue?.id?.toLowerCase()
```
before running it multiple times inside the filter with both sides of the `===` lower-cased

```suggestion
      ({ isSaved, venueId }) =>
        isSaved && venueId?.toLowerCase() === currentVenueId
```